dt-backend/src/model/user/user.service.ts

```diff
@@ -12,4 +24,37 @@ export class UserService {
       },
     });
   }
+
+  async createUser(data: Prisma.UserCreateInput): Promise<User> {
+    return prisma.user.create({
+      data,
+      include: {
+        audit: true,
+        empire: true,
+      },
+    })
+  }
+
+  async updateUser(params: {where: Prisma.UserWhereUniqueInput; data: Prisma.UserUpdateInput;}): Promise<User> {
+    const { data, where } = params;
+    return prisma.user.update({
+      data,
+      where,
+      include: {
+        audit: true,
+        empire: true,
+      }
+    })
+  }
+
+  async deleteUser(where: Prisma.UserWhereUniqueInput): Promise<User> {
```

Comment:
I won't mention it from now on - but param/variable naming is important^^!

Reply:
oh i gotcha. even though the types already exist somewhere in some autogenerated prisma file, you'd rather us make our own using DTO's.

Reply:
Yup - you'll see later why )).